.github/workflows/labeler.yml

```
@@ -0,0 +1,25 @@
+name: Labeler
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)